tracing-forest/src/prelude.rs

```diff
@@ -0,0 +1,39 @@
+//! The module re-exports a number of useful components that you may wish
+//! to consume.
+
+pub use tracing::Instrument;
+
+pub use tracing::{
+    debug,
+    debug_span,
+    error,
+    error_span,
+    event,
+    info,
+    info_span,
+    span,
+    trace,
+    trace_span,
+    warn,
+    warn_span,
+};
+
+// As these macros are macro_export, they are published at the crate root.
+/*
+#[cfg(feature = "uuid")]
+pub use crate::{
+    uuid_debug_span,
+    uuid_error_span,
+    uuid_info_span,
+    uuid_span,
+    uuid_trace_span,
+    uuid_warn_span,
+};
+*/
```

Comment:
Can we just define them in this module and have them publicly accessible? If so I can do that

Reply:
I'd rather leave that to you since you may know better the implications of that .